token-lending/js/package.json

```diff
@@ -1,72 +1,55 @@
-  "types": "lib/index.d.ts",
-  "files": [
-    "lib"
-  ],
-  "scripts": {
-    "build": "rollup -c rollup.config.ts",
-    "build:program": "cargo build-bpf --manifest-path ../program/Cargo.toml",
-    "start": "ts-node cli/main.ts",
-    "start-with-test-validator": "start-server-and-test 'solana-test-validator --bpf-program LendZqTs7gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi ../../target/deploy/spl_token_lending.so --reset --quiet' http://localhost:8899/health start",
-    "lint": "eslint --ext .ts {cli,client}/* && prettier --check \"{cli,client}/**/*.ts\"",
-    "lint:fix": "eslint --ext .ts {cli,client}/* --fix && prettier --write \"{cli,client}/**/*.ts\"",
-    "cluster:localnet": "rm -f .env",
-    "cluster:devnet": "cp cluster-devnet.env .env",
-    "cluster:testnet": "cp cluster-testnet.env .env",
-    "cluster:mainnet-beta": "cp cluster-mainnet-beta.env .env"
-  },
-  "dependencies": {
-    "@solana/spl-token": "0.1.6",
-    "@solana/web3.js": "^1.20.0",
-    "bn.js": "^5.2.0",
-    "mkdirp": "^1.0.4"
-  },
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "^19.0.0",
-    "@tsconfig/recommended": "^1.0.1",
-    "@types/bn.js": "^5.1.0",
-    "@types/eslint": "^7.2.13",
-    "@types/eslint-plugin-prettier": "^3.1.0",
-    "@types/mkdirp": "^1.0.1",
-    "@types/mz": "^2.7.3",
-    "@types/node": "^15.12.5",
-    "@types/prettier": "^2.3.0",
-    "@types/rollup-plugin-json": "^3.0.2",
-    "@typescript-eslint/eslint-plugin": "^4.28.0",
-    "@typescript-eslint/parser": "^4.28.1",
-    "dotenv": "^10.0.0",
-    "eslint": "^7.29.0",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-plugin-prettier": "^3.4.0",
-    "prettier": "^2.2.1",
-    "rollup": "^2.52.3",
-    "rollup-plugin-json": "^4.0.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-sourcemaps": "^0.6.3",
-    "rollup-plugin-typescript2": "^0.30.0",
-    "start-server-and-test": "^1.12.5",
-    "ts-node": "^10.0.0",
-    "typescript": "^4.3.4"
-  },
-  "engines": {
-    "node": ">= 10"
-  }
+    "name": "@solana/spl-token-lending",
+    "version": "0.2.1",
+    "description": "SPL Token Lending JavaScript API",
+    "license": "MIT",
+    "author": "Solana Maintainers <maintainers@solana.foundation>",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/solana-labs/solana-program-library"
+    },
+    "bugs": {
+        "url": "https://github.com/solana-labs/solana-program-library/issues"
+    },
+    "keywords": [],
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "files": [
+        "lib"
+    ],
+    "scripts": {
+        "build": "rm -rf lib/* && tsc",
+        "deploy": "gh-pages -d docs -e token-lending",
+        "docs": "rm -rf docs/* && typedoc",
+        "lint": "eslint . --ext .ts && prettier --check '**/*.{ts,js,json}'",
+        "lint:fix": "eslint . --ext .ts --fix && prettier --write '**/*.{ts,js,json}'"
+    },
+    "dependencies": {
```

Comment:
Can you add a dependency on `buffer-layout` here?

Reply:
Good catch, done.